extensions/blocks/podcast-player/view.js

```diff
@@ -0,0 +1,66 @@
+/* global _wpmejsSettings, MediaElementPlayer, jetpackPodcastPlayers */
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const playerInstances = {};
+const meJsSettings = typeof _wpmejsSettings !== undefined ? _wpmejsSettings : {};
```

Comment:
`typeof varToTest` always returns a STRING, so

```es6
typeof _wpmejsSettings !== undefined
```

... will be always `True`. we should use `"undefined"` or

```es6
_wpmejsSettings !== undefined`
```

I prefer using `typeof`.

Reply:
Silly me! I will update this instant!

Reply:
fixed